tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -259,7 +259,8 @@ def class_host_ft(
         ).create()
         yield host
         skip_yum_update_during_provisioning(template='Kickstart default finish', reverse=True)
-        host.delete()
+        with satelliteSetting("destroy_vm_on_host_delete=True"):
```

Comment:
single quotes are nicer !
```suggestion
        with satelliteSetting('destroy_vm_on_host_delete=True'):
```